src/Conan.cmake

```diff
@@ -6,7 +6,7 @@ macro(run_conan)
   if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
     message(STATUS "Downloading conan.cmake from https://github.com/conan-io/cmake-conan")
     file(
-      DOWNLOAD "https://raw.githubusercontent.com/conan-io/cmake-conan/0.17.0/conan.cmake"
+      DOWNLOAD "https://raw.githubusercontent.com/conan-io/cmake-conan/0.18.1/conan.cmake"
       "${CMAKE_BINARY_DIR}/conan.cmake"
       EXPECTED_HASH SHA256=3bef79da16c2e031dc429e1dac87a08b9226418b300ce004cc125a82687baeef
```

Comment:
This hash needs to be changed

Reply:
They removed the hash from the README file in the latest version [18.1](https://github.com/conan-io/cmake-conan/commit/df0f64a0920eab35033b17cc1104aa8b37af204f?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5). I don't know why but I ran it on my PC to find the new hash. I hope it works this time and fixes my clang-cl issue on VS2022. Then I plan to update the project_options version of the [cmake_conan_boilerplate_template](https://github.com/cpp-best-practices/cmake_conan_boilerplate_template)

Reply:
Unfortunately, it didn't work. OK NVM, I have to use the canonical Conan workflow with the "CmakeDeps" and "CmakeToolchain" generators.